kupong_integration.gemspec

```diff
@@ -23,6 +23,5 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 1.14"
   spec.add_development_dependency "rake", "~> 10.0"
-  spec.add_development_dependency "rspec", "~> 3.0"
```

Comment:
Cool, no worries. We should definitely have some test coverage if we plan to publish this.